Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -57,7 +58,8 @@ struct GeneratorCLI: AsyncParsableCommand {
       try await generatorTask.value
     }
 
-    print("\nTime taken for this generator run: \(elapsed.intervalString).")
+    logger.info("")
+    logger.info("Time taken for this generator run: \(elapsed.intervalString).")
```

Comment:
If we're sticking to structured logging, we should omit formatting trivia. Also, use of string interpolation makes it hard to process such logs (grouping messages, aggregating timing values for benchmarks etc), these values should be logged as metadata:
```suggestion
    logger.info("Generator run finished successfully", metadata: ["generatorRunInterval": .string(elapsed.intervalString)])
```

Reply:
@euanh WDYT of using swift-log metadata here? I'm myself somewhat unsure if we should stick to machine-readable metadata instead of human-readable output, but I know metadata is actually preferred when using swift-log. Maybe we could achieve both, somehow defaulting to the human-readable option?

Reply:
I've had similar thoughts.   It's nice to have the structured logs, particularly for things which are running in CI, but I sometimes find the default swift-log output a bit out of place in an interactive command.     I'd like to have a 'human-readable' configuration option for swift-log but I'm not aware of one.

In this particular case, I think `elapsedTime`, `totalRuntime` or similar might be a better name than `generatorRunInterval`

Reply:
I think if we can configure human-readable logging without all the extra attributes for swift-log that would be beneficial. It's harder to read with the date, label, and module name....

Also wonder if this couldn't just be a separate PR. I wouldn't mind splitting this off and starting a separate PR with all prints replaced with logging, and use of metadata for structured logs, and even having a stab at cleanup up the swift-log output...

Reply:
Done. I did leave the block in the EndToEndTests:

```swift
  // Cleanup
  logger.info("Removing SDK to cleanup...")
  try await Shell.run("swift experimental-sdk remove \(bundleName)")
```

But as you can see, there is no need to update these lines with structure as is required elsewhere.